integration

```diff
@@ -96,6 +96,7 @@
   "bacco007/sensor.waternsw",
   "barban-dev/homeassistant-midea-dehumidifier",
   "barleybobs/homeassistant-ecowater-softener",
+  "basbruss/adaptive-cover",
```

Comment:
https://github.com/basbruss/adaptive-cover/actions/runs/11426903428/job/31790406035 hassfest is failing.

Reply:
That's linked to this PR to core https://github.com/home-assistant/core/pull/128965

Reply:
New release removes code that failed hassfest